src/httpserver.rs

```diff
@@ -209,6 +209,23 @@ fn valid_github_username(n: &str) -> bool {
     n.chars().all(|c| c.is_ascii_alphanumeric() || c == '-')
 }
 
+/// Is `n` a valid GitHub repository name?
```

Comment:
Did you get these rules by supplying an invalid reponame as you did for user names?

Reply:
Yep, same process. I should probably add a comment in to that effect. Fixed in https://github.com/softdevteam/snare/pull/39/commits/3c177aa7433e8a0a6576ffa12ae25f4d70d0bce6.